# Vietnamese translation for apport
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-12-07 16:35+0100\n"
"PO-Revision-Date: 2011-03-27 11:11+0000\n"
"Last-Translator: Nguyễn Anh <Unknown>\n"
"Language-Team: Vietnamese <vi@li.org>\n"
"Language: vi\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2022-12-07 14:36+0000\n"
"X-Generator: Launchpad (build 41e7553f8097ce3a683f90e835708cbe7bf0006c)\n"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr ""

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr ""

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr ""

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid "Please enter your password to access problem reports of system programs"
msgstr ""

#: ../apport/ui.py:165
msgid "This package does not seem to be installed correctly"
msgstr ""

#: ../apport/ui.py:175
#, python-format
msgid ""
"This does not seem to be an official %s package. Please retry after updating "
"the indexes of available packages, if that does not work then remove related "
"third party packages and try again."
msgstr ""

#: ../apport/ui.py:205
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"Bạn có vài gói phiên bản cũ. Vui lòng nâng cấp những gói sau và kiểu tra nếu "
"lỗi vẫn xuất hiện:\n"
"\n"
"%s"

#: ../apport/ui.py:359
msgid "unknown program"
msgstr "chương trình không xác định"

#: ../apport/ui.py:362
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "Xin lỗi, chương trình \"%s\" đã kết thúc bất ngờ"

#: ../apport/ui.py:365 ../apport/ui.py:1899
#, python-format
msgid "Problem in %s"
msgstr "Vấn đề trong %s"

#: ../apport/ui.py:370
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"Máy tính của bạn không đủ bộ nhớ trống để tự động phân tích vấn đề và gửi "
"báo cáo tới các nhà phát triển."

#: ../apport/ui.py:430 ../apport/ui.py:448 ../apport/ui.py:621
#: ../apport/ui.py:630 ../apport/ui.py:867 ../apport/ui.py:1677
#: ../apport/ui.py:1860 ../apport/ui.py:1865
msgid "Invalid problem report"
msgstr "Báo cáo vấn đề không hợp lệ"

#: ../apport/ui.py:431
msgid "You are not allowed to access this problem report."
msgstr "Bạn không được phép truy cập báo cáo vấn đề này."

#: ../apport/ui.py:439
msgid "Error"
msgstr "Lỗi"

#: ../apport/ui.py:441
msgid "There is not enough disk space available to process this report."
msgstr "Không đủ dung lượng đĩa để thực hiện báo cáo này."

#: ../apport/ui.py:479
msgid "No PID specified"
msgstr ""

#: ../apport/ui.py:481
msgid "You need to specify a PID. See --help for more information."
msgstr ""

#: ../apport/ui.py:492 ../apport/ui.py:597
msgid "Invalid PID"
msgstr "PID không hợp lệ"

#: ../apport/ui.py:493
msgid "The specified process ID does not exist."
msgstr ""

#: ../apport/ui.py:498
msgid "Not your PID"
msgstr ""

#: ../apport/ui.py:499
msgid "The specified process ID does not belong to you."
msgstr ""

#: ../apport/ui.py:554
msgid "No package specified"
msgstr "Chưa gói nào được xác định"

#: ../apport/ui.py:556
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"Bạn cần xác định một gói hoặc một PID. Xem --help để biết thêm thông tin."

#: ../apport/ui.py:584
msgid "Permission denied"
msgstr "Không đủ quyền truy cập"

#: ../apport/ui.py:586
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"Tiến trình không phải của bạn. Vui lòng chạy chương trình với tài khoản của "
"chủ tiến trình hoặc với tài khoản quản trị root"

#: ../apport/ui.py:599
msgid "The specified process ID does not belong to a program."
msgstr "ID của tiến trình không phải của một chương trình."

#: ../apport/ui.py:623
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr ""
"Trình tìm triệu chứng %s không xác định được gói phần mềm đã bị ảnh hưởng"

#: ../apport/ui.py:631
#, python-format
msgid "Package %s does not exist"
msgstr "Gói %s không tồn tại"

#: ../apport/ui.py:660 ../apport/ui.py:872 ../apport/ui.py:905
#: ../apport/ui.py:915
msgid "Cannot create report"
msgstr "Không tạo được bản báo cáo"

#: ../apport/ui.py:675 ../apport/ui.py:734 ../apport/ui.py:752
msgid "Updating problem report"
msgstr "Cập nhật lại vấn đề đang báo cáo"

#: ../apport/ui.py:677
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"Bạn không phải là người báo hoặc người theo dõi vấn đề này, hay là báo cáo "
"này đã bị báo trùng hoặc đã bị đóng.\n"
"Hãy tạo một tạo một báo lỗi mới sử dụng \"apport-bug\"."

#: ../apport/ui.py:689
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"Bạn không phải là người báo cáo vấn đề này. Một cách dễ dàng hơn để đánh dấu "
"lỗi bị trùng với một cái khác là di chuyển những lời giải thích và đính kèm "
"của bạn vào một lỗi mới.\n"
"Sau đó, chúng tôi khuyên bạn tạo một báo cáo lỗi mới sử dụng \"Trình báo "
"lỗi\" và tạo một chú giải về vấn đề mà bạn báo.\n"
"Bạn có thật sự muốn thực hiện?"

#: ../apport/ui.py:735 ../apport/ui.py:753
msgid "No additional information collected."
msgstr "Không có thêm thông tin đã được thu tập"

#: ../apport/ui.py:816
msgid "What kind of problem do you want to report?"
msgstr "Bạn muốn báo cáo vấn đề loại nào?"

#: ../apport/ui.py:837
msgid "Unknown symptom"
msgstr "Dấu hiệu không xác định"

#: ../apport/ui.py:838
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "Không biết dấu hiệu \"%s\""

#: ../apport/ui.py:874
msgid ""
"The window option cannot be used on Wayland.\n"
"\n"
"Please find the window's process ID and then run 'ubuntu-bug <process ID>'.\n"
"\n"
"The process ID can be found by running the System Monitor application. In "
"the Processes tab, scroll until you find the correct application. The "
"process ID is the number listed in the ID column."
msgstr ""

#: ../apport/ui.py:890
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""
"Sau khi đóng thông điệp này vui lòng nhấp vào một cửa sổ ứng dụng để báo cáo "
"một vấn đề về nó."

#: ../apport/ui.py:907 ../apport/ui.py:916
msgid "xprop failed to determine process ID of the window"
msgstr "xprop không thể định số ID tiến trình của cửa sổ"

#: ../apport/ui.py:931
#, python-format
msgid "%(prog)s <report number>"
msgstr ""

#: ../apport/ui.py:932
msgid "Specify package name."
msgstr ""

#: ../apport/ui.py:938 ../apport/ui.py:1070
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr "Thêm một thẻ cho báo cáo. Có thể thực hiện nhiều lần."

#: ../apport/ui.py:976
#, python-format
msgid ""
"%(prog)s [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""

#: ../apport/ui.py:987
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"Chạy với chế độ lọc lỗi. Yêu cầu hoặc --package và một tùy chọn --pid, hoặc "
"chỉ là --pid. Nếu không cách nào trong hai cách trên được sử dụng, hiển thị "
"danh sách những dấu hiệu đã biết. (Mặc định nếu một tham số được đưa ra.)"

#: ../apport/ui.py:996
msgid "Click a window as a target for filing a problem report."
msgstr ""

#: ../apport/ui.py:1005
msgid "Start in bug updating mode. Can take an optional --package."
msgstr "Bắt đầu ở chế độ cập nhật lỗi. Có thể dùng tuỳ chọn --package."

#: ../apport/ui.py:1014
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""
"Đệ trình báo cáo lỗi về một triệu chứng. (Mặc định nếu tên triệu chứng là "
"tham số duy nhất.)"

#: ../apport/ui.py:1023
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"Xác định tên gói trong chế độ --file-bug. Tùy chọn này nếu một --pid được "
"xác định. (Mặc định nếu tên gói được đưa ra như là tham số duy nhất.)"

#: ../apport/ui.py:1034
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""

#: ../apport/ui.py:1042
msgid "The provided pid is a hanging application."
msgstr ""

#: ../apport/ui.py:1050
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"Báo cáo lỗi sử dụng các tệp .apport hoặc .crash thay vì sử dụng các tệp tạm "
"trong %s. (Mặc định nếu tệp tin là tham số duy nhất.)"

#: ../apport/ui.py:1060
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""

#: ../apport/ui.py:1078
msgid "Print the Apport version number."
msgstr "In ra số phiên bản Trình báo lỗi"

#: ../apport/ui.py:1240
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr ""

#: ../apport/ui.py:1244
msgid "Run gdb session"
msgstr ""

#: ../apport/ui.py:1245
msgid "Run gdb session without downloading debug symbols"
msgstr ""

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:1247
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr ""

#: ../apport/ui.py:1309
msgid "Can't remember send report status settings"
msgstr ""

#: ../apport/ui.py:1313
msgid ""
"Saving crash reporting state failed. Can't set auto or never reporting mode."
msgstr ""

#: ../apport/ui.py:1396 ../apport/ui.py:1409
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr "Báo cáo vấn đề này gắn với chương trình đã bị gỡ."

#: ../apport/ui.py:1433
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr ""

#: ../apport/ui.py:1494 ../apport/ui.py:1618 ../apport/ui.py:1869
msgid "This problem report is damaged and cannot be processed."
msgstr "Báo cáo vấn đề này bị hỏng và không thể được thực hiện"

#: ../apport/ui.py:1503
msgid "This report is about a package that is not installed."
msgstr ""

#: ../apport/ui.py:1511
msgid "An error occurred while attempting to process this problem report:"
msgstr ""

#: ../apport/ui.py:1528
msgid ""
"You have two versions of this application installed, which one do you want "
"to report a bug against?"
msgstr ""

#: ../apport/ui.py:1533
#, python-format
msgid "%s snap"
msgstr ""

#: ../apport/ui.py:1534
#, python-format
msgid "%s deb package"
msgstr ""

#: ../apport/ui.py:1575
#, python-format
msgid "%s is provided by a snap published by %s. Contact them via %s for help."
msgstr ""

#: ../apport/ui.py:1580
#, python-format
msgid ""
"%s is provided by a snap published by %s. No contact address has been "
"provided; visit the forum at https://forum.snapcraft.io/ for help."
msgstr ""

#: ../apport/ui.py:1679
msgid "Could not determine the package or source package name."
msgstr "Không thể xác định tên của gói hoặc gói nguồn."

#: ../apport/ui.py:1704
msgid "Unable to start web browser"
msgstr "Không thể bật trình duyệt web"

#: ../apport/ui.py:1705
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "Không thể bật trình duyệt web để mở %s."

#: ../apport/ui.py:1815
msgid "Network problem"
msgstr "Lỗi mạng"

#: ../apport/ui.py:1819
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""
"Không thể kết nối tới cơ sở lỗi, vui lòng kiểm tra kết nối Internet của bạn."

#: ../apport/ui.py:1851
msgid "Memory exhaustion"
msgstr "Hết bộ nhớ"

#: ../apport/ui.py:1853
msgid "Your system does not have enough memory to process this crash report."
msgstr "Hệ thống của bạn không đủ bộ nhớ để thực hiện báo cáo lỗi này."

#: ../apport/ui.py:1904
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"Không thể báo cáo vấn đề:\n"
"\n"
"%s"

#: ../apport/ui.py:1960 ../apport/ui.py:1972
msgid "Problem already known"
msgstr "Vấn đề đã biết"

#: ../apport/ui.py:1962
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"Vấn đề đã được báo cáo trong báo cáo lỗi hiển thị trong trình duyệt web. Vui "
"lòng kiểm tra nếu bạn có thể thêm bất cứ thông tin gì có ích cho những người "
"phát triển."

#: ../apport/ui.py:1974
msgid "This problem was already reported to developers. Thank you!"
msgstr ""

#: ../bin/apport-cli.py:84
msgid "Press any key to continue..."
msgstr "Nhấn phím bất kì để tiếp tục..."

#: ../bin/apport-cli.py:91
msgid "What would you like to do? Your options are:"
msgstr "Bạn muốn làm gì? Các lựa chọn là:"

#: ../bin/apport-cli.py:104
#, python-format
msgid "Please choose (%s):"
msgstr "Vui lòng chọn (%s):"

#: ../bin/apport-cli.py:172
#, python-format
msgid "(%i bytes)"
msgstr "(%i bytes)"

#: ../bin/apport-cli.py:174 ../gtk/apport-gtk.py:169 ../kde/apport-kde.py:406
msgid "(binary data)"
msgstr "(dữ liệu nhị phân)"

#: ../bin/apport-cli.py:209 ../gtk/apport-gtk.py:208 ../gtk/apport-gtk.ui.h:6
#: ../kde/apport-kde.py:199
msgid "Send problem report to the developers?"
msgstr "Gửi báo cáo vấn đề tới những người phát triển?"

#: ../bin/apport-cli.py:211
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"Sau khi báo cáo về vấn đề được gửi, vui lòng điền vào mẫu trong trang\n"
"web vừa tự động mở ra."

#: ../bin/apport-cli.py:218
#, python-format
msgid "&Send report (%s)"
msgstr "&Gửi báo cáo (%s)"

#: ../bin/apport-cli.py:223
msgid "&Examine locally"
msgstr ""

#: ../bin/apport-cli.py:227
msgid "&View report"
msgstr "&Xem báo cáo"

#: ../bin/apport-cli.py:230
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr "G&iữ tệp báo cáo để gửi sau hoặc sao chép"

#: ../bin/apport-cli.py:235
msgid "Cancel and &ignore future crashes of this program version"
msgstr "Hủy và bỏ &qua hỏng hóc của chương trình này những lần sau"

#: ../bin/apport-cli.py:238 ../bin/apport-cli.py:332 ../bin/apport-cli.py:366
#: ../bin/apport-cli.py:389
msgid "&Cancel"
msgstr "&Thôi"

#: ../bin/apport-cli.py:273
msgid "Problem report file:"
msgstr "Tệp báo cáo vấn đề:"

#: ../bin/apport-cli.py:279 ../bin/apport-cli.py:284
msgid "&Confirm"
msgstr "&Xác nhận"

#: ../bin/apport-cli.py:283
#, python-format
msgid "Error: %s"
msgstr "Lỗi: %s"

#: ../bin/apport-cli.py:289 ../kde/apport-kde.py:462
msgid "Collecting problem information"
msgstr "Thu thập thông tin về vấn đề"

#: ../bin/apport-cli.py:291
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"Thông tin được thu thập có thể gửi tới những người phát triển để cải tiến\n"
"ứng dụng. Công việc này có thể mất vài phút."

#: ../bin/apport-cli.py:307 ../gtk/apport-gtk.ui.h:17 ../kde/apport-kde.py:511
msgid "Uploading problem information"
msgstr "Tải lên thông tin về vấn đề"

#: ../bin/apport-cli.py:309
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"Thông tin thu thập đang được gửi tới hệ thống kiểm duyệt lỗi.\n"
"Côn việc này có thể mất vài phút."

#: ../bin/apport-cli.py:365
msgid "&Done"
msgstr "Đã &xong"

#: ../bin/apport-cli.py:371
msgid "none"
msgstr "không"

#: ../bin/apport-cli.py:373
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Đã chọn: %s. Nhiều lựa chọn:"

#: ../bin/apport-cli.py:390
msgid "Choices:"
msgstr "Các lựa chọn:"

#: ../bin/apport-cli.py:404
msgid "Path to file (Enter to cancel):"
msgstr "Đường đẫn tới tệp (Enter để hủy):"

#: ../bin/apport-cli.py:410
msgid "File does not exist."
msgstr "Tệp không tồn tại."

#: ../bin/apport-cli.py:412
msgid "This is a directory."
msgstr "Nó là một thư mục"

#: ../bin/apport-cli.py:418
msgid "To continue, you must visit the following URL:"
msgstr "Để tiếp tục, bạn phải vào địa chỉ URL sau:"

#: ../bin/apport-cli.py:421
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"Giờ bạn có thể chạy một trình duyệt, hoặc chép địa chỉ URL này vào một trình "
"duyệt hoặc máy tính khác."

#: ../bin/apport-cli.py:427
msgid "Launch a browser now"
msgstr "Chạy một trình duyệt"

#: ../bin/apport-cli.py:443
msgid "No pending crash reports. Try --help for more information."
msgstr "Không có báo cáo lỗi nào đang chờ. Thử --help để biết thêm thông tin."

#: ../bin/apport-retrace.py:44
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr "Không đưa dấu vết mới vào báo cáo, nhưng in chúng ra stdout."

#: ../bin/apport-retrace.py:53
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"Bắt đầu một phiên gỡ lỗi tương tác với gdb sử dụng các thông tin lưu lại "
"được khi chương trình gặp lỗi. (-o ignored: không viết lại báo cáo)"

#: ../bin/apport-retrace.py:62
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr "Ghi báo cáo đã sửa đổi tới tệp đưa ra thay vì thay đổi báo cáo gốc"

#: ../bin/apport-retrace.py:72
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr ""
"Xóa các thông tin lưu lại được khi chương trình gặp lỗi khỏi báo cáo sau khi "
"đã ghi lại vết của ngăn xếp."

#: ../bin/apport-retrace.py:80
msgid "Override report's CoreFile"
msgstr "Ghi đè CoreFile của báo cáo"

#: ../bin/apport-retrace.py:86
msgid "Override report's ExecutablePath"
msgstr "Ghi đè ExecutablePath của báo cáo"

#: ../bin/apport-retrace.py:92
msgid "Override report's ProcMaps"
msgstr "Ghi đè ProcMaps của báo cáo"

#: ../bin/apport-retrace.py:98
msgid "Rebuild report's Package information"
msgstr "Tạo lại thông tin gói của báo cáo"

#: ../bin/apport-retrace.py:105
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""

#: ../bin/apport-retrace.py:119
msgid ""
"Build another temporary sandbox for installing gdb and its dependencies "
"using the same release as the report rather than whatever version of gdb you "
"have installed."
msgstr ""

#: ../bin/apport-retrace.py:129
msgid "Report download/install progress when installing packages into sandbox"
msgstr ""

#: ../bin/apport-retrace.py:136
msgid "Prepend timestamps to log messages, for batch operation"
msgstr ""

#: ../bin/apport-retrace.py:142
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr ""

#: ../bin/apport-retrace.py:150
msgid "Cache directory for packages downloaded in the sandbox"
msgstr ""

#: ../bin/apport-retrace.py:156
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""

#: ../bin/apport-retrace.py:167 ../bin/apport-valgrind.py:97
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr ""

#: ../bin/apport-retrace.py:174
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"Đường dẫn đến tệp với thông tin xác thực cho cơ sở dữ liệu về những sự cố. "
"Múc đích là tạo ra một ID để tải lên những dấu vết đã theo (chỉ khi nào "
"không dùng -g, -o hoặc -s)"

#: ../bin/apport-retrace.py:184
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"Hiển thị ngăn xếp dấu vết dò lại được và yêu cầu xác nhận trước khi gửi "
"chúng tới cơ sở dữ liệu lỗi."

#: ../bin/apport-retrace.py:192
msgid "Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr ""
"Đường dẫn tới bản sao cơ sở dữ liệu sqlite (mặc định: không kiểm tra bản sao)"

#: ../bin/apport-retrace.py:200
msgid "Do not add StacktraceSource to the report."
msgstr ""

#: ../bin/apport-retrace.py:213
msgid "You cannot use -C without -S. Stopping."
msgstr ""

#. translators: don't translate y/n,
#. apport currently only checks for "y"
#: ../bin/apport-retrace.py:243
msgid "OK to send these as attachments? [y/n]"
msgstr "Đồng ý gửi chúng như những đính kèm? [y/n]"

#: ../bin/apport-unpack.py:29
#, python-format
msgid "%(prog)s <report> <target directory>"
msgstr ""

#: ../bin/apport-unpack.py:31
msgid "Report file to unpack"
msgstr ""

#: ../bin/apport-unpack.py:33
msgid "directory to unpack report to"
msgstr ""

#: ../bin/apport-unpack.py:59
msgid "Destination directory exists and is not empty."
msgstr "Thư mục đích tồn tại và không rỗng."

#: ../bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr ""

#: ../bin/apport-valgrind.py:46
msgid "specify the log file name produced by valgrind"
msgstr ""

#: ../bin/apport-valgrind.py:52
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""

#: ../bin/apport-valgrind.py:60
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""

#: ../bin/apport-valgrind.py:69
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""

#: ../bin/apport-valgrind.py:78
msgid "report download/install progress when installing packages into sandbox"
msgstr ""

#: ../bin/apport-valgrind.py:86
msgid ""
"the executable that is run under valgrind's memcheck tool for memory leak "
"detection"
msgstr ""

#: ../bin/apport-valgrind.py:129
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr ""

#: ../data/apportcheckresume.py:68
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr ""

#: ../data/apportcheckresume.py:73
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr ""

#: ../data/apportcheckresume.py:81
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"Quá trình khôi phục bị dừng khi sắp kết thúc và sẽ xuất hiện để hoàn thành "
"bình thường."

#: ../data/kernel_oops.py:30
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""
"Hệ thống của bạn có thể trở nên không ổn định bây giờ và cần khởi động lại."

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde-mime.desktop.in.h:1
#: ../kde/apport-kde.desktop.in.h:1
msgid "Report a problem..."
msgstr "Báo cáo một vấn đề..."

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde-mime.desktop.in.h:2
#: ../kde/apport-kde.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "Báo cáo một trục trặc tới nhà phát triển"

#: ../gtk/apport-gtk.py:185
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr ""

#: ../gtk/apport-gtk.py:188
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr ""

#: ../gtk/apport-gtk.py:193 ../kde/apport-kde.py:214 ../kde/apport-kde.py:266
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr ""

#: ../gtk/apport-gtk.py:216 ../gtk/apport-gtk.ui.h:14 ../kde/apport-kde.py:207
msgid "Send"
msgstr ""

#: ../gtk/apport-gtk.py:234 ../gtk/apport-gtk.py:666 ../gtk/apport-gtk.ui.h:11
#: ../kde/apport-kde.py:330
msgid "Show Details"
msgstr ""

#: ../gtk/apport-gtk.py:247 ../kde/apport-kde.py:258 ../kde/apport-kde.py:276
msgid "Continue"
msgstr ""

#: ../gtk/apport-gtk.py:271
#, python-format
msgid "The application %s has stopped responding."
msgstr ""

#: ../gtk/apport-gtk.py:275
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr ""

#: ../gtk/apport-gtk.py:292 ../kde/apport-kde.py:222
#, python-format
msgid "Package: %s"
msgstr ""

#: ../gtk/apport-gtk.py:298 ../kde/apport-kde.py:229
msgid "Sorry, a problem occurred while installing software."
msgstr ""

#: ../gtk/apport-gtk.py:310 ../gtk/apport-gtk.py:334 ../kde/apport-kde.py:238
#, python-format
msgid "The application %s has experienced an internal error."
msgstr ""

#: ../gtk/apport-gtk.py:316 ../kde/apport-kde.py:245
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr ""

#: ../gtk/apport-gtk.py:347 ../gtk/apport-gtk.ui.h:7 ../kde/apport-kde.py:271
msgid "If you notice further problems, try restarting the computer."
msgstr ""

#: ../gtk/apport-gtk.py:352 ../kde/apport-kde.py:278
msgid "Ignore future problems of this type"
msgstr ""

#: ../gtk/apport-gtk.py:670 ../kde/apport-kde.py:327
msgid "Hide Details"
msgstr ""

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:552 ../kde/apport-kde.py:595
msgid "Apport"
msgstr ""

#: ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr ""

#: ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr ""

#: ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr ""

#: ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr ""

#: ../gtk/apport-gtk.ui.h:8
msgid "Remember this in future"
msgstr ""

#: ../gtk/apport-gtk.ui.h:9
msgid "Ignore future problems of this program version"
msgstr ""

#: ../gtk/apport-gtk.ui.h:10
msgid "Relaunch this application"
msgstr ""

#: ../gtk/apport-gtk.ui.h:12
msgid "_Examine locally"
msgstr ""

#: ../gtk/apport-gtk.ui.h:13
msgid "Don't send"
msgstr ""

#: ../gtk/apport-gtk.ui.h:15
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>Thu thập thông tin về vấn đề</b></big>"

#: ../gtk/apport-gtk.ui.h:16
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"Thông tin đang được thu thập, nó có thể giúp các nhà phát triển sửa vấn đề "
"bạn báo cáo."

#: ../gtk/apport-gtk.ui.h:18
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>Tải lên thông tin về vấn đề</b></big>"

#: ../gtk/apport-gtk.ui.h:19 ../kde/apport-kde.py:513
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"Thông tin thu thập được đang được gửi tới hệ thống kiểm soát lỗi. Công việc "
"này có thể mất vài phút."

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Báo cáo tệp lỗi"

#: ../kde/apport-kde.py:261
msgid "Leave Closed"
msgstr ""

#: ../kde/apport-kde.py:262 ../kde/apport-kde.py:432
msgid "Relaunch"
msgstr ""

#: ../kde/apport-kde.py:355
msgid "Username:"
msgstr "Tài khoản:"

#: ../kde/apport-kde.py:356
msgid "Password:"
msgstr "Mật khẩu:"

#: ../kde/apport-kde.py:461
msgid "Collecting Problem Information"
msgstr ""

#: ../kde/apport-kde.py:464
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""

#: ../kde/apport-kde.py:510
msgid "Uploading Problem Information"
msgstr ""
